### DISCUSSION TITLE
Anybody got a line on getting the HD Model pack supported? (I know I'm disgusting)

### DISCUSSION THREAD

**Discussion by Implicit-D** [sultim-t/xash-rt#141] [Q&A]
I'm curious about support for the HD model pack that comes with the Steam release. I'm one of those heretics that actually kind of likes them better. It's a nostalgia thing. I grew up reading Gamepro and seeing the Dreamcast and PS2 port previews back in the day before I had a PC that could run Half-Life worth a damn. So of course I got it for PS2. And when I got a PC and a DC, of course I got the leaked DC HL1 as soon as I could. It was at the very beginning of my computer/code tinkering phase so I've always had a soft spot for the HD model pack.

**Comment by saarstar15** [comment]
I can maybe forgive someone if they grew up on the ps2...

Yokes aside. I've had no issues playing with model replacement mods, other than a few of the special materials that sultim added to entities, like Barney's metallic helmet, will no longer be present. 

So you can try adding the HD pack model files to this path from the HL root folder:

rt/valve/models/

The HD pack sprites can be added as well:

rt/valve/sprites/

Cheers,
